.github/workflows/ci-quality.yml

```diff
@@ -0,0 +1,52 @@
+name: CI — quality
```

Comment:
Why the name `ci-quality` here? The file is doing pre-commit linting. Could we change the name to reflect what the workflow is doing?